rcl/test/CMakeLists.txt

```diff
@@ -61,7 +61,10 @@ function(test_target_function)
   )
 
   # TODO(wjwwood): remove this when the graph API works properly for more than OpenSplice.
-  if(rmw_implementation STREQUAL "rmw_opensplice_cpp")
+  if(
+    rmw_implementation STREQUAL "rmw_opensplice_cpp"
+    OR rmw_implementation STREQUAL "rmw_connext_cpp"
```

Comment:
I changed the logic (596c0ec9faaa81cb4ba9dc064c27ba340412c023) and opened issues to track why:

- Fast RTPS: https://github.com/eProsima/ROS-RMW-Fast-RTPS-cpp/issues/64
- Connext Dynamic: https://github.com/ros2/rmw_connext/issues/203